src/overlays/actors/ovl_En_Ru2/z_en_ru2.c

```diff
@@ -186,7 +215,7 @@ CsCmdActorCue* EnRu2_GetCue(PlayState* play, s32 cueChannel) {
     return NULL;
 }
 
-s32 func_80AF27D0(EnRu2* this, PlayState* play, u16 cueId, s32 cueChannel) {
+s32 EnRu2_CheckCueMatchingID(EnRu2* this, PlayState* play, u16 cueId, s32 cueChannel) {
```

Comment:
Current codebase style prefers `Id` over `ID` when camel casing. Same goes for function below